### PR TITLE
Reduce number of redirect urls

### PR DESCRIPTION
**Pull request by grahambancroft**
## Done

Add 'catch all' redirects for insights, content, news and devices

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- Make sure the old redirects work with the new versions

Fixes https://github.com/ubuntudesign/web-squad/issues/239

### CODE REVIEW

File: redirects.yaml [hunk 1]
```diff
@@ -1429,17 +1089,6 @@ system/files/metal_sales_v3\.pdf/?: "https://insights.ubuntu.com/"
...
-tablet/backed-by-canonical/?: "/partners/backed-by-canonical"
-tablet/operat/?: "/mobile"
-tablet/operators-and-oems/?: "/mobile/partners"
-tablet/register/?: "https://docs.google.com/a/canonical.com/forms/d/1VKTn1u2CifkTk7lQQC0Ws4tC7yaCep4DjRp34S6cV14/viewform"
-tablet/?: "/mobile"
-tablet/features/?: "/mobile/features"
-tablet/devices/?: "/mobile/devices"
-tablet/developers/?: "/mobile/developers"
-tablet/partners/?: "/mobile/partners"
-tablet/contact-us/?: "/mobile/contact-us"
```
Comment (anthonydillon) [inline redirects.yaml]: Shouldn't phone and tablet urls be added to the delected.yaml?

Comment (grahambancroft) [reply redirects.yaml]: I've added them as redirects to /mobile which is already in the deleted.yaml, feels like less code?

Comment (anthonydillon) [reply redirects.yaml]: Fair enough, thanks

File: redirects.yaml [hunk 2]
```diff
@@ -1,39 +1,24 @@
...
-2013/05/08/people-behind-ubuntu-quality-howard-2/?: "https://insights.ubuntu.com"
-2013/07/edge-founders/?: "https://insights.ubuntu.com/2013/08/22/ubuntu-edge-thank-you/"
-2013/07/press-pack-ubuntu-edge/?: "https://insights.ubuntu.com/press-centre/"
-2014/02/press-pack-mwc-2014/?: "https://insights.ubuntu.com/2014/02/01/mwc-2014-online-press-pack/"
-2014/03/vmware-openstack-webinar/?: "https://insights.ubuntu.com/2014/03/18/customer-stories-from-openstackvmware-projects/"
+2009/.+/?: "https://insights.ubuntu.com"
+2010/.+/?: "https://insights.ubuntu.com"
+2011/.+/?: "https://insights.ubuntu.com"
+2012/.+/?: "https://insights.ubuntu.com"
+2013/.+/?: "https://insights.ubuntu.com"
```
Comment (anthonydillon) [inline redirects.yaml]: We need `2014` to catch: 
- 2014/02/press-pack-mwc-2014/
- 2014/03/vmware-openstack-webinar/

File: redirects.yaml [hunk 3]
```diff
@@ -1,39 +1,24 @@
...
+tablet/.+/?: "/mobile"
```
Comment (anthonydillon) [inline redirects.yaml]: http://www.ubuntu.com-pr-2433.run.demo.haus/tablet is 404'ing

File: redirects.yaml [hunk 4]
```diff
@@ -1,39 +1,24 @@
...
+phone/.+/?: "/mobile"
```
Comment (anthonydillon) [inline redirects.yaml]: This one is also just 404'ing

File: redirects.yaml (hunk 2 above)
Comment (pmahnke) [reply redirects.yaml]: that's my fault

the '?' makes it a non-greedy search... so remove the ? and it should work?

```
2009/.+/: "https://insights.ubuntu.com"
2010/.+/: "https://insights.ubuntu.com"
2011/.+/: "https://insights.ubuntu.com"
2012/.+/: "https://insights.ubuntu.com"
2013/.+/: "https://insights.ubuntu.com"
2014/.+/: "https://insights.ubuntu.com"
```

*Note: there might be more*

Comment (grahambancroft) [reply redirects.yaml]: Fixed

**Comment by anthonydillon** [comment]
The deleted yaml is now to greedy

**Comment by grahambancroft** [comment]
@anthonydillon fixed

**Comment by anthonydillon** [comment]
Sorry, still not working.

**Comment by grahambancroft** [comment]
Fixed

**Comment by grahambancroft** [comment]
@anthonydillon I've added the code review label back as the code changed a fair bit

**Comment by anthonydillon** [comment]
Could you `rebase` this down to a single commit with a descriptive message. Thanks.

**Pull request merged**